README.md

```diff
@@ -22,11 +22,15 @@ Add the following configuration to `build.gradle`:
             classpath 'com.palantir:jacoco-coverage:<version>'
         }
     }
-    
+
+    apply plugin: 'java'
     apply plugin: 'jacoco-coverage'
     jacocoCoverage {
-        threshold 0.5  // Enfore minimum code coverage of 50% for every file, class, package, and project.
-        whitelist "MyClass.java"  // Exempt files named MyClass.java from coverage requirements.
+        // Enfore minimum code coverage of 50% for every Java file
+        file 0.5
+
+        // Whitelist files named MyClass.java from coverage requirements.
+        file 0.0, "MyClass.java"
```

Comment:
yep. syntax is now `file 0.0, "MyClass.java"` or `_class 0.0, "org/package/MyClass"`